packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -4,40 +4,43 @@
 *--------------------------------------------------------------------------------------------*/
 
 .presentation-instance-filter-dialog {
-  min-width: 50vw;
+  .presentation-instance-filter-dialog-content-container {
+    min-height: calc(var(--iui-size-2xl) * 5 + var( --iui-size-xs));
+    min-width: calc(var(--iui-size-2xl) * 8);
 
-  .presentation-instance-filter-title {
-    margin-bottom: 0;
-    padding: 6px;
-  }
+    .presentation-instance-filter-title {
+      margin-bottom: 0;
+      padding: 6px;
+    }
 
-  .presentation-instance-filter-content {
-    max-height: 80vh;
-    overflow: hidden;
+    .presentation-instance-filter-content {
+      max-height: 80vh;
+      overflow: hidden;
 
-    .presentation-property-filter-builder {
-      overflow: scroll;
-      max-height: 70vh;
+      .presentation-property-filter-builder {
+        overflow: scroll;
```

Comment:
Looks like vertical scrollbar is shown sometimes when there is no need for it (probably it takes into account possible size of horizontal scrollbar). I think this can be set to `auto`

Reply:
fixed